docs/reference/changelog-r2020.md

```diff
@@ -28,6 +28,7 @@ Released on XXX.
     - Fixed Matlab API.
     - Fixed missing stdout/stderr flush when a controller is changed or restarted while simulation is running (thanks to tsampazk).
     - Fixed the ROS controller of the Universal Robots UR3e, UR5e and UR10e to send the success state when a trajectory succeeded (thanks to Tim-Stoll).
+    - Fixed a crash when trying to add IndexedLineSet to a geometry's shape node with no coordinates.
```

Comment:
```suggestion
    - Fixed a crash when trying to add an [IndexedLineSet](indexedlineset.md) node to a geometry's [Shape](shape.md) node with no coordinates.
```

Sorry for being picky.

Reply:
I am also not sure about, `coordinates`, should we remove the `s`?

Reply:
I would leave it plural, see [this](http://www.englishlessonsbrighton.co.uk/followed-singular-plural-noun/).